docs/en/enterprise-edition/policy-reference/ibm-policies/ibm-iam-policies/bc-ibm-2-3.adoc

```diff
@@ -0,0 +1,47 @@
+== IBM Cloud API key creation is not restricted in account settings in Terraform
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/ibm/IBM_RestrictAPIkeyCreationInAccountSettings.yaml[CKV2_IBM_3]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to ensure that the creation of API keys is restricted within the account settings. Allowing unrestricted creation of API keys can pose a security risk because it increases the chances of unauthorized access to the system. By restricting API key creation, you minimize the potential for keys to be created without oversight, thereby maintaining better control over who has access to your resources and reducing the risk of accidental or malicious misuse.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* ibm_iam_account_settings
+* *Arguments:* restrict_create_platform_apikey
+
+By setting `restrict_create_platform_apikey` to `RESTRICTED`, API key creation is restricted in the account settings. This ensures that only authorized processes and users can create API keys, thereby reducing the risk of unauthorized access to your IBM Cloud resources.
+
+In this example, the `ibm_iam_account_settings` resource is used to configure the account settings and restrict API key creation by setting `restrict_create_platform_apikey`` to `RESTRICTED`.
```

Comment:
In this example, the 'restrict_create_platform_apikey' attribute is set to 'RESTRICTED' in the 'ibm_iam_account_settings' resource to restrict API key creation.